p_seq: [6, 0, 0, 0, 0, 0] # Goal position x1, y1, z1, x2, y2, z2 ...

yea_seq: [0, 0] # Goal angle theta1, theta2, theta3 ...
